react/features/welcome/components/BlankPage.js

```diff
@@ -1,13 +1,21 @@
 import PropTypes from 'prop-types';
-import { Component } from 'react';
+import React, { Component } from 'react';
+import { ActivityIndicator, View } from 'react-native';
```

Comment:
Ah, `BlankPage` is technically used on both Web and react-native (though practically it should currently only be visible on react-native) so webpack will fail to bundle the Web and that's why automatic PR testing fails.

Reply:
Oh, I'll take care of that.